Buildkite.TestAnalytics.Common/Payload.fs

```diff
@@ -44,7 +46,9 @@ module Payload =
                   Branch = optionalEnvVar "BUILDKITE_BRANCH"
                   CommitSha = optionalEnvVar "BUILDKITE_COMMIT"
                   Message = optionalEnvVar "BUILDKITE_MESSAGE"
-                  Url = optionalEnvVar "BUILDKITE_BUILD_URL" }
+                  Url = optionalEnvVar "BUILDKITE_BUILD_URL"
+                  Version = "0.1.1"
```

Comment:
I really want to extract this value from the .fsproj file, but can't quite see the correct way. I've tried things like `System.Reflection.AssemblyVersionAttribute` and `Assembly.GetEntryAssembly().GetName().Version` but no luck...